test-integration/student/dashboard.js

```diff
@@ -62,7 +62,7 @@ describe('Student Dashboard', function(){
     expect(unstartedEvents).not.to.be.empty;
 
     // that's all folks
-    this.student.logout();
+    // this.student.logout();
 
```

Comment:
Ah, I `xit`'d this because the script times out when students have like a ton of assignments because of the nested selenium calls. Was going to see if you knew of a _quicker_ way to write the `findVisibleEvents`